src/AspNetCoreModuleV2/InProcessRequestHandler/inprocessapplication.cpp

```diff
@@ -365,23 +365,35 @@ IN_PROCESS_APPLICATION::SetEnvironementVariablesOnWorkerProcess(
     VOID
 )
 {
-    HRESULT hr = S_OK;
+    auto variables = m_pConfig->QueryEnvironmentVariables();
```

Comment:
Let's file a follow up issue to remove the old style hash table here (as it is making this needlessly complex).